src/Gui/SoDatumLabel.cpp

```diff
@@ -59,6 +59,63 @@ using namespace Gui;
 
 // ------------------------------------------------------
 
+
+namespace glVec3f{
+
+void glVertex(SbVec3f pt){
+    glVertex3f(pt[0], pt[1], pt[2]);
+}
+
+void glVertexes(std::vector<SbVec3f> pts){
```

Comment:
This should be a const reference to avoid copying the data for every function call.